appveyor.yml

```diff
@@ -4,6 +4,7 @@ environment:
   GOPATH: c:\gopath
 stack: go 1.14
 before_test:
+  - choco install mingw
```

Comment:
is this a necessity for `-race` ?

Reply:
yes, it needs gcc :/

Reply:
https://ci.appveyor.com/project/code-ready/crc/builds/37279990